src/context/AppProvider/AppContext.tsx

```diff
@@ -99,25 +94,6 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     require(`dayjs/locale/${selectedLocale}.js`)
   }, [selectedLocale])
 
-  /**
-   * handle wallet disconnect/switch logic
-   */
-  useEffect(() => {
```

Comment:
now that we don't need to clear portfolio and tx history, this effect would just be used to set the wallet id in the store.

given that, we can just set it in the wallet provider instead. this gives us the additional benefit of persisting the previously connected wallet id, allowing the rest of the app context to preload everything the portfolio needs. 

once we have https://github.com/shapeshift/web/issues/3327 to delay prompting for the password until we need to unlock the wallet to sign, we should have near instant page loads, bar some market data requests (which we can cache in a future pr too).